src/Http/Http.Abstractions/src/IHttpContextAccessor.cs

```diff
@@ -3,8 +3,14 @@
 
 namespace Microsoft.AspNetCore.Http
 {
+    /// <summary>
+    /// Accessor to the <see cref="HttpContext"/>
+    /// </summary>
     public interface IHttpContextAccessor
     {
+        /// <summary>
+        /// Gets or sets the associated <see cref="HttpContext"/>.
```

Comment:
```suggestion
        /// Gets or sets the current <see cref="HttpContext"/>. Returns <see langword="null" /> if there is no active <see cref="HttpContext" />.
```

Reply:
Only the `<summary>` shows up in IntelliSense, which is why I put the "Returns" section here instead of a `<returns>` element. Feel free to copy it there as well for completeness.